### ISSUE TITLE
Loop using for 

### ISSUE THREAD

**Issue by Qamar-93** [facg3/FindYourGame#18]
https://github.com/facg3/FindYourGame/blob/74be8cd3c4ba0c86dccf80e3e8d341a645328c2f/src/handlers.js#L54
if you want to loop over an array and return one value, you can use reduce instead of for 
it will give you better performance :smile:

**Comment by MMortaga** [comment]
You're right. We got a bunch of changes to do before our presentation, so thanks for pointing this out.
